release/release.go

```diff
@@ -1108,17 +1113,42 @@ func uploadRelease(v version.Version) {
 	}
 }
 
-var linuxRepoVersionsStable = map[string]string{
-	"development": "4.0.0-15-gabcde123", // any non-rc pre-release version will send the package to the "development" repo
-	"testing":     "4.0.0-rc0",          // any rc version will send the package to the "testing" repo
-	"4.3":         "4.3.0",              // any 4.3 stable release version will send the package to the "4.3" repo
-	"4.4":         "4.4.0",              // any 4.4 stable release version will send the package to the "4.4" repo
-	"4.9":         "4.9.0",              // any 4.9 stable release version will send the package to the "4.9" repo
-	"5.0":         "5.0.0",              // any 5.0 stable release version will send the package to the "5.0" repo
+type VersionWithSigningInfo struct {
+	mongoVersionNumber string
+	notaryKeyName string
+	notaryToken string
 }
 
-var linuxRepoVersionsUnstable = map[string]string{
-	"development": "4.0.0-15-gabcde123", // any non-rc pre-release version will send the package to the "development" repo
+var linuxRepoVersionsStable = map[string]VersionWithSigningInfo{
+	"development": {"4.0.0-15-gabcde123", "", ""}, // any non-rc pre-release version will send the package to the "development" repo
+	"testing":     {"4.0.0-rc0", "", ""},          // any rc version will send the package to the "testing" repo
+	"4.3":         {"4.3.0", "server-4.4", os.Getenv("NOTARY_TOKEN_4_4")},              // any 4.3 stable release version will send the package to the "4.3" repo
+	"4.4":         {"4.4.0", "server-4.4", os.Getenv("NOTARY_TOKEN_4_4")},              // any 4.4 stable release version will send the package to the "4.4" repo
+	"4.9":         {"4.9.0", "server-5.0", os.Getenv("NOTARY_TOKEN_5_0")},              // any 4.9 stable release version will send the package to the "4.9" repo
```

Comment:
based on what I'm seeing [here](https://www.mongodb.org/static/pgp/), it looks like there is actually a separate key for 4.9. Interestingly, there _doesn't_ appear to be a separate key for 4.3.

We might want to confirm the following with @briananthonymccarthy:
- what key to use for the 4.3 repo
- no key used for development and testing repos

Reply:
I am checking with Brian on these two questions.

Reply:
After talking with @zakhark , we figured 
1. There is no need to publish to 4.3 repo. It's a dev release repo and the Build team never advertised it. There are only unstable server versions inside.  I guess we were initially used it to test the separate release of Tools from the Server.  
So I removed 4.3 from versionsToRelease.
2. development and testing repos are not advertised either. No one is expected to use them with package manager. So it doesn't matter which key was used to sign them. However, since the notary key and token cannot be absent when calling curator, I set `server-4.4`key for these two repos.

Reply:
Confirmed with Zahkar and Kelsey, 4.6-4.9 repos are not needed, so I removed 4.9 from here as well.